OWNERS_ALIASES

```diff
@@ -9,10 +9,9 @@ aliases:
     - Tafhim
     - Tof1973
   srep-functional-leads:
-    - rafael-azevedo
     - mjlshen
     - iamkirkbater
```

Comment:
```suggestion
    - clcollins
```

Reply:
As requested
* moved @iamkirkbater to TL
* added @clcollins to FL